.infrastructure/helm-chart/todoapp/templates/rbac.yml

```diff
@@ -0,0 +1,30 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: {{ .Values.todoapp.RBAC.name }}
+  namespace: {{ .Values.todoapp.namespace }}
```

Comment:
Ensure that the `todoapp.namespace` value is defined in the `values.yaml` file. This is crucial for the correct deployment of the ServiceAccount in the specified namespace.